CHANGELOG.md

```diff
@@ -0,0 +1,108 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+----
```

Comment:
Can you add the `----` separator between versions please. This is used to build latest changelogs

Reply:
Whoa! I did not realize that. Will do.

Reply:
I added the `----` horizontal separators, Luis!